rdr_service/model/participant_summary.py

```diff
@@ -73,7 +73,11 @@
 _COMPUTE_RETENTION_ELIGIBLE_SQL = """
     CASE WHEN
       consent_for_study_enrollment = 1
-      AND (consent_for_electronic_health_records = 1 OR consent_for_dv_electronic_health_records_sharing = 1)
+      AND (
+        consent_for_electronic_health_records = 1 OR
+        ehr_consent_expire_status = 2 OR
+        consent_for_dv_electronic_health_records_sharing = 1
+      )
```

Comment:
Shouldn't participants remain retention eligible even if they've changed their EHR consent from Yes to No?

Reply:
Good catch, I will make the change.

Reply:
Done

Reply:
👍